models/voltmeter.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ *  voltmeter.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "voltmeter.h"
+
+// Includes from nestkernel:
+#include "event_delivery_manager_impl.h"
+
+namespace nest
+{
+Voltmeter::Voltmeter()
+  : Multimeter()
```

Comment:
Since the merge of #1345, both `Multimeter` and `Voltmeter` start with a lower case letter. Please adapt all occurrences accordingly.